stable/nats/templates/statefulset.yaml

```diff
@@ -29,10 +29,15 @@ spec:
         {{- if .Values.podLabels }}
 {{ toYaml .Values.podLabels | indent 8 }}
         {{- end }}
-      {{- if .Values.podAnnotations }}
+{{- if or .Values.podAnnotations .Values.metrics.enabled }}
       annotations:
+{{- if or .Values.podAnnotations }}
```

Comment:
should this rather be
`{{- if .Values.podAnnotations }}`

Reply:
Thanks. Fixed